protocol/avdl/keybase1/git.avdl

```diff
@@ -16,10 +16,46 @@ protocol git {
   @typedef("string")
   record RepoID {}
 
-  // Right now, the only metadata in our locally-encrypted-then-server-stored
-  // blob is the name of the repo. Everything else (mtime, last editing user)
-  // is inferred by the server based on who's doing the push, and stored in the
-  // clear.
+  enum GitLocalMetadataVersion {
+    V1_1,
+    V2_2,
+    V3_3,
+    V4_4,
+    V5_5,
+    V6_6,
+    V7_7,
+    V8_8,
+    V9_9,
+    V10_10
+  }
```

Comment:
nice versioning! Would you remind me why we have these explicit unsupported things rather than just not listing them yet?

Reply:
I'm copying the scheme we used for `HeaderPlaintext` in the chat protocol. I agree that it's weird to pay a space cost for not-yet-invented future versions, but I haven't asked questions yet. @mmaxim do you remember why chat did it this way?